src/main/java/com/rocketden/main/game_object/Game.java

```diff
@@ -13,9 +14,10 @@
 
     private Room room;
 
-    private Map<String, Player> players;
+    // TODO discussion: is the key userId or nickname or none?
+    private Map<String, Player> players = new HashMap<>();
```

Comment:
Generally, I think players are going to be identified by `userId`, both now and in the future. Although there is the concern of exposing these values to the users, which leads to security issues, I think that if we ever allow duplicate usernames in the same game (or just across the platform, may be a concern), or accounts with specific settings, `userId` will be the best.

Not sure whether this needs to be a Map or just a List -- though I think eventually this will be useful when attempting to find a user quickly, so I'd say a Map with `userId` key and `Player` value is the best design for now. I would also be fine with just a List though.

Reply:
Sounds good, I updated the code to use userId (previously was using nickname)